app/views/moderations/index.html.erb

```diff
@@ -62,7 +62,7 @@
   </center>
   <div class="container">
     <% @articles.each do |article| %>
-      <% @rating_vote = article.rating_votes.where(user_id: current_user.id).first %>
+      <% @rating_vote = @rating_votes.where(article: article).first %>
```

Comment:
this trick is to avoid too many queries. In the controller we load all rating votes of the articles of interest for the user, and here we filter that query with the subset pertaining the article in the loop